llvm/lib/Object/OffloadBinary.cpp

```diff
@@ -23,14 +24,20 @@
 #include "llvm/Object/IRObjectFile.h"
 #include "llvm/Object/ObjectFile.h"
 #include "llvm/Support/Alignment.h"
+#include "llvm/Support/BinaryStreamReader.h"
 #include "llvm/Support/FileOutputBuffer.h"
 #include "llvm/Support/SourceMgr.h"
+#include "llvm/Support/Timer.h"
 
 using namespace llvm;
 using namespace llvm::object;
 
 namespace {
 
+static llvm::TimerGroup
+    ClangOffloadBundlerTimerGroup("Clang Offload Bundler Timer Group",
```

Comment:
We could rename this "OffloadBundlerTimerGroup".

I've been thinking about the newly seperated bundler APIs as the "Offload Bundler", and the clang-offload-bundler binary tool as the "Clang Offload Bundler"

Reply:
Good point.  This code is no longer in Clang, so no need for that in the name.

Reply:
resolved in tlatest patch